src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/history.html

```diff
@@ -29,6 +29,7 @@
                             <pre>{{ value.message|linebreaksbr }}</pre>
                         {% endfor %}
                     </p>
+                    <a href="{% url 'plugins:sendmail:send' organizer=request.event.organizer.slug event=request.event.slug %}?from_log={{ log.pdata.mailid }}">Send mail</a>
```

Comment:
This should be translatable, e.g. ``{% trans "Send a new email based on this" %}``